app/src/main/res/layout/screen_patient_search_results.xml

```diff
@@ -17,6 +17,47 @@
     app:title="Anish Acharya with a really long surname"
     app:titleTextColor="@color/black" />
 
+  <RelativeLayout
+    android:id="@+id/patientsearchresults_empty_state"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:layout_above="@+id/patientsearchresults_new_patient_container"
+    android:layout_below="@+id/patientsearchresults_toolbar"
+    android:paddingBottom="@dimen/spacing_16"
+    android:visibility="gone"
+    tools:visibility="visible">
+
+    <TextView
+      android:id="@+id/patientsearchresults_empty_state_text"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_centerInParent="true"
+      android:layout_marginEnd="@dimen/spacing_24"
+      android:layout_marginStart="@dimen/spacing_24"
+      android:text="@string/patientsearchresults_empty_state"
+      android:textAppearance="@style/Clinic.V2.TextAppearance.H6.Grey2" />
+
+    <View
```

Comment:
Isn't there a widget named `Space` that's generally meant for this purpose?

Reply:
This is not empty state. It's the tail of the arrow. 😄